Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.cxx

```diff
@@ -155,6 +159,13 @@ void qMRMLSubjectHierarchyComboBox::setMRMLScene(vtkMRMLScene* scene)
 }
 
 //------------------------------------------------------------------------------
+void qMRMLSubjectHierarchyComboBox::clearCurrentItem()
```

Comment:
Please call this function also clearSelection. I don't see a reason for this to have a name different from the one in the tree view when we only call that.

Reply:
Because treeviews, models etc. deal with **selected** items/indexes : http://doc.qt.io/qt-4.8/qtreeview.html#selectedIndexes

While a combobox only deals with **current** item/index, not "selections" : 
http://doc.qt.io/qt-4.8/qcombobox.html#currentIndex-prop

Thoughts?

Reply:
This function right now is the most basic wrapper imaginable for TreeView::clearSelection, as the only thing it does is calls that. Having a different name for such a wrapper is I think misleading.

Also I think the name clearCurrentItem is not straightforward, as the first thing that occurs to me is that this function clears the _content_ of the current item.

I really think clearSelection is better in every way.

Reply:
sounds good, changing that.